src/main/java/OOPExercise1/Triangle.java

```diff
@@ -0,0 +1,112 @@
+package OOPExercise1;
+
+import java.util.Arrays;
+
+public class Triangle {
+
+    private double x1;
+    private double y1;
+
+    private double x2;
+    private double y2;
+
+    private double x3;
+    private double y3;
+
+
+    public Triangle() {
+
+    }
+
+    public Triangle(double x1, double y1, double x2, double y2, double x3, double y3) {
+
+        this.x1 = x1;
+        this.y1 = y1;
+
+        this.x2 = x2;
+        this.y2 = y2;
+
+        this.x3 = x3;
+        this.y3 = y3;
+
+    }
+
+
+    public double edgeTriangle(double x1, double y1, double x2, double y2) {
```

Comment:
Hàm này hoạt động không phụ thuộc vào các thành phần thuộc object nên nó bắt buộc phải là một static method

```suggestion
    public static double getDistance(double x1, double y1, double x2, double y2) {
```

Ngoài ra, tên hàm cũng chưa thực sự hợp lý lắm.

Reply:
done!